OPTIONS.md

```diff
@@ -9,8 +9,8 @@
 <h5>Default value:</h4><pre><code>[]</pre></code><h4>continueOnAttach</h4><p>If true, we&#39;ll automatically resume programs launched and waiting on <code>--inspect-brk</code></p>
 <h5>Default value:</h4><pre><code>false</pre></code><h4>customDescriptionGenerator</h4><p>Customize the textual description the debugger shows for objects (local variables, etc...). Samples:<br>      1. this.toString() // will call toString to print all objects<br>      2. this.customDescription ? this.customDescription() : defaultValue // Use customDescription method if available, if not return defaultValue<br>      3. function (def) { return this.customDescription ? this.customDescription() : def } // Use customDescription method if available, if not return defaultValue<br>      </p>
 <h5>Default value:</h4><pre><code>undefined</pre></code><h4>customPropertiesGenerator</h4><p>Customize the properties shown for an object in the debugger (local variables, etc...). Samples:<br>    1. { ...this, extraProperty: &#39;12345&#39; } // Add an extraProperty 12345 to all objects<br>    2. this.customProperties ? this.customProperties() : this // Use customProperties method if available, if not use the properties in this (the default properties)<br>    3. function () { return this.customProperties ? this.customProperties() : this } // Use customDescription method if available, if not return the default properties<br><br>    Deprecated: This is a temporary implementation of this feature until we have time to implement it in the way described here: <a href="https://github.com/microsoft/vscode/issues/102181">https://github.com/microsoft/vscode/issues/102181</a></p>
-<h5>Default value:</h4><pre><code>undefined</pre></code><h4>cwd</h4><p>Absolute path to the working directory of the program being debugged.</p>
```

Comment:
This file is autogenerated from `strings.ts` -- can you update the doc there?

Reply:
Can you point me to this file? I don't see it

Reply:
The `strings.ts` file doesn't have `node.attach.cwd` so i don't know where that bit is coming from?
i only see `node.launch.cwd` and `browser.launch.cwd`

Reply:
strings.ts is consumed in generate-contributions.ts, which is what the docs are actually created from. node.launch.cwd is shared in there between launch and attach -- the string naming is sort of a mess, in part because it contains strings copied from the previous debug adapters which were not consistent.